kairon/shared/auth.py

```diff
@@ -189,6 +179,8 @@ def authenticate(username: Text, password: Text):
                 detail="Incorrect username or password",
                 headers={"WWW-Authenticate": "Bearer"},
             )
+        UserActivityLogger.add_log(account=user.get('account'), a_type=UserActivityType.login.value, email=user.get('email'))
```

Comment:
remove this as this loggin is already happening inside generate_login_tokens

Reply:
move UserActivityLogger.is_login_within_cooldown_period(user.get('email')) inside generate_login_tokens() at line 190